File: Famoser.MassPass.Api/src/Models/Request/SyncRequest.php
<?php
/**
 * Created by PhpStorm.
 * User: famoser
 * Date: 22/05/2016
 * Time: 23:01
 */

namespace Famoser\MassPass\Models\Request;


use Famoser\MassPass\Models\Request\Base\ApiRequest;

class SyncRequest extends ApiRequest
{
    /**
     * @var Entities\RefreshEntity[] $RefreshEntities
     */
    public $RefreshEntities;
    /**
     * @var string[] $CollectionIds
     */
    public $CollectionIds;
}

File: Famoser.MassPass.Data/Entities/Communications/Request/SyncRequest.cs
using System;
using System.Collections.Generic;
using Famoser.MassPass.Data.Entities.Communications.Request.Base;
using Famoser.MassPass.Data.Entities.Communications.Request.Entities;

namespace Famoser.MassPass.Data.Entities.Communications.Request
{
    public class SyncRequest : ApiRequest
    {
        public SyncRequest()
        {
            RefreshEntities = new List<RefreshEntity>();
            CollectionIds = new List<Guid>();
        }

        public List<RefreshEntity> RefreshEntities { get; set; }
        public List<Guid> CollectionIds { get; set; }
    }
}
